controller-app/src/main/java/frc/nemesis/controller/ControllerApp.java

```diff
@@ -267,36 +323,61 @@ private void updateSideButtons() {
   }
 
   private void updatePendingCommand() {
-    if (selectedLevel != null && selectedSide != null && selectedDirection != null) {
-      pendingCommand = selectedDirection + "_" + selectedSide + "_" + selectedLevel;
+    if (selectedSide != null && selectedDirection != null) {
```

Comment:
@shashankmadala there has been a good amount of changes in this code in main recently.  can you create a new branch from main, and apply your changes in as minimal a manner as possile?  I do not want to change the logic around adding the item to network tables, or refreshing from the same.